### PR TITLE
edge: fix dіrectionality attributes

### PR DESCRIPTION
**Pull request by fippo**
fixes an issue with the direction attribute and unskips three tests.

### CODE REVIEW

File: src/js/edge/edge_shim.js [hunk 1]
```diff
@@ -656,13 +656,16 @@ var edgeShim = {
...
 
-              track = rtpReceiver.track;
-              receiverList.push([track, rtpReceiver]);
-              // FIXME: not correct when there are multiple streams but that is
-              // not currently supported in this shim.
-              stream.addTrack(track);
+                track = rtpReceiver.track;
+                receiverList.push([track, rtpReceiver]);
+                // FIXME: not correct when there are multiple streams but that
+                // is not currently supported in this shim.
```
Comment (KaptenJansson) [inline src/js/edge/edge_shim.js:666]: What happens in this case? Throw an error or does it silently ignore the other streams?

Comment (fippo) [reply src/js/edge/edge_shim.js:666]: I have a branch for unified plan support which also improves the behaviour here

Comment (KaptenJansson) [reply src/js/edge/edge_shim.js:666]: I see

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**